plugins/module_utils/data_set.py

```diff
@@ -92,6 +92,10 @@ class DataSet(object):
 
     _VSAM_UNCATALOG_COMMAND = " DELETE '{0}' NOSCRATCH"
 
+    MVS_PARTITIONED = frozenset({"PE", "PO", "PDSE", "PDS"})
+    MVS_SEQ = frozenset({"PS", "SEQ", "BASIC"})
+    MVS_VSAM = frozenset({"KSDS", "ESDS", "RRDS", "LDS", "VSAM"})
```

Comment:
Not an issue but why use a frozenset?

Reply:
Not sure what their reason was but given a frozen set is an immutable set maybe they just did not want anyone to alter them after creation.

Reply:
To avoid accidentally adding or removing elements, that could cause problems between calls of the class' methods.

Reply:
Thanks for the clarification @rexemin 👍